### PR TITLE
Flicker bug

### PR DESCRIPTION
**Pull request by johnosullivan**

### CODE REVIEW

File: StreetTagged/Components/Cells/PhotoCell.swift [hunk 1]
```diff
@@ -27,11 +28,17 @@ class PhotoCell: BaseCollectionViewCell {
...
+    public func setImage(imageURL: String) {
+        let url = URL(string: imageURL)
+        imageView.kf.setImage(with: url)
     }
     
     public func getImage() -> UIImage? {
         return imageView.image
     }
+    
+    public func resetImage() {
```
Comment (dconroy) [inline StreetTagged/Components/Cells/PhotoCell.swift]: why are we removing reset here?

File: StreetTagged/Components/Cells/PostCell.swift [hunk 2]
```diff
@@ -320,6 +326,14 @@ class PostCell: BaseCollectionViewCell {
             isPostBookmarked = !isPostBookmarked
         }
     }
+    
+    override func prepareForReuse() {
+       super.prepareForReuse()
+
+       print("prepareForReuse")
```
Comment (dconroy) [inline StreetTagged/Components/Cells/PostCell.swift]: do we want this in the PR?

Comment (johnosullivan) [reply StreetTagged/Components/Cells/PostCell.swift]: Keeping

File: StreetTagged/Components/Cells/PhotoCell.swift (hunk 1 above)
Comment (johnosullivan) [reply StreetTagged/Components/Cells/PhotoCell.swift]: Removed

File: StreetTagged/Components/Cells/PostCell.swift [hunk 3]
```diff
@@ -333,10 +343,15 @@ extension PostCell: UICollectionViewDelegateFlowLayout, UICollectionViewDelegate
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: cellID, for: indexPath) as! PhotoCell
         if let post = post {
-            if indexPath.item == 0 {
-                cell.imageView.loadImage(post.image)
+            if (isResetting) {
```
Comment (dconroy) [inline StreetTagged/Components/Cells/PostCell.swift]: are we still using the isResetting?

**Review by dconroy — APPROVED** [review]

**Pull request merged**